test/test_wpad.cc

```diff
@@ -39,7 +39,8 @@ TEST(wpad, dhcp) {
     EXPECT_NE(localent, nullptr);
 
     // Create network adapter
-    net_adapter_s adapter = {0};
+    net_adapter_s adapter;
+    memset(&adapter, 0, sizeof(adapter));
```

Comment:
What is inner struct in that type? I suppose it was a warning about aggregate initialization.
It should be
```c++
net_adapter_s adapter{};
```

Reply:
Fixed.